pipelines/experimental/gitops/build-push-promote-task.yaml

```diff
@@ -163,18 +163,19 @@ spec:
   - name: promote
     securityContext:
       privileged: true
-    image: quay.io/redhat-developer/gitops-cli:v0.1
+    image: quay.io/redhat-developer/gitops-cli:latest
```

Comment:
Would this be latest or digest value as well like others.

Reply:
@kvijai82 confirmed this is taken care in next PR

Reply:
Will be updating this whole task to match the one in events in the next RC.